plugins/scaffolder-backend/src/scaffolder/tasks/StorageTaskBroker.ts

```diff
@@ -135,6 +144,9 @@ export class StorageTaskBroker implements TaskBroker {
   ) {}
   private deferredDispatch = defer();
 
+  /**
+   * @inheritdoc
```

Comment:
hm that's awesome if it can build like that - i thought i had to do it on this form'

```
  /**
   * {@inheritdoc CatalogApi.getEntityAncestors}
   */
```